openshift/README.md

```diff
@@ -60,6 +60,13 @@ If SELinux is in enforcing mode, it is recommend to grant [the `spc_t` type][6]
 * `volumes: hostPath`: Accesses the Docker socket and the host's `proc` and `cgroup` folders, for metric collection.
 * `SELinux type: spc_t`: Accesses the Docker socket and all processes' `proc` and `cgroup` folders, for metric collection. You can read more about this type [in this Red Hat article][6].
 
+**Note**: The Docker socket is owned by the root group, so you may need to elevate the Agent's privileges to pull in Docker metrics. To allow the Agent process proper pe3rmissions, update your SCC with the below:
```

Comment:
small typo - "pe3rmissions"

Reply:
@DylanLovesCoffee can you suggest @cswatt a better wording for "To allow the Agent process proper permissions"? Something along "Running the [Agent Process](https://docs.datadoghq.com/integrations/process/) as root is necessary in newer versions of Openshift in order to use Docker socket to collect the required metrics from the host"

Reply:
That's a good alternative, and so far this looks good to me. If we want to be more explicit, I would add to what we already have with:
>The Docker socket is owned by the root group, so you may need to elevate the Agent's privileges to pull in Docker metrics. To run the Agent Process as a root user, you can configure your SCC with the following:

Reply:
@DylanLovesCoffee - Is this ready to be merged?